src/components/dashboard/dashboard.tsx

```diff
@@ -7,16 +7,15 @@ import { useAtom } from 'jotai'
 import { useRouter } from 'next/navigation'
 import peanut from '@squirrel-labs/peanut-sdk'
 
+import trapezoid from '@/assets/icons/trapezoid.svg'
+
 import * as global_components from '@/components/global'
```

Comment:
This import isn't used

In general, is it a good idea to import everything with *? Or should we import specific values as needed? Would this improve page speed?